Sources/UIColor+Sweet.swift

```diff
@@ -21,6 +21,17 @@ public extension UIColor {
         }
     }
 
+    /// Convenience initializers for RGB colors.
+    ///
+    /// - Parameters:
+    ///   - red: The red part.
+    ///   - green: The green part.
+    ///   - blue: The blue part.
+    ///   - alpha: The alpha part.
+    public convenience init(r red: Double, g green: Double, b blue: Double, a alpha: Double = 100) {
```

Comment:
From both the method signature and from documentation this looks exactly the same as the regular RGBA init method, that uses percentage instead. 🤔